lab-jessica/routes/book-routes.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const storage = require('../lib/storage.js');
+const Book = require('../model/book.js');
+
+module.exports = function(router){
+
+  router.get('/api/books', function(req, res) {
+    res.status(400).send('bad request');
+  });
+
+  router.get('/api/books/:id', function(req, res) {
```

Comment:
You're sending a 400 bad request if someone neglects to pass an ID as part of the url, so you shouldn't need to test for it using an if conditional here in this get route.